src/components/CoSign.js

```diff
@@ -7,22 +7,21 @@ import {get} from 'api';
 import Body from 'components/Body';
 import ErrorMessage from 'components/ErrorMessage';
 import Loader from 'components/Loader';
-import LoginButton, {LoginButtonIcon} from 'components/LoginButton';
 import {ConfigContext, SubmissionContext} from 'Context';
-import {Toolbar, ToolbarList} from 'components/Toolbar';
 import Types from 'types';
-
 import {getBEMClassName} from 'utils';
+import LoginOptionsDisplay from 'components/LoginOptions/LoginOptionsDisplay';
 
 // TODO: tests!
 
 const getCosignStatus = async (baseUrl, submissionUuid) => {
   const endpoint = `${baseUrl}submissions/${submissionUuid}/co-sign`;
-  const response = await get(endpoint);
-  return response;
+  return await get(endpoint);
 };
 
 const CoSignAuthentication = ({form, submissionUuid, saveStepData, authPlugin}) => {
+  const config = useContext(ConfigContext);
```

Comment:
the idea is nice, however this won't work because the formio component calls `ReactDOM.render` and I don't think this shares the context from the top-level react root (our structure is `react root -> formio root -> react root` and the react roots are independent).

The display config should be passed down in `FormStep.js` to the formio config, which the `src/formio/components/CoSign.js` should then use in the `ConfigContext.Provider` in the `renderReact` method.